src/SwedbankPay.Sdk/ResourceBase.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Logging;
+
+using SwedbankPay.Sdk.Exceptions;
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk
+{
+    public abstract class ResourceBase
+    {
+        protected readonly SwedbankPayOptions swedbankPayOptions;
+        internal SwedbankPayHttpClient swedbankPayClient;
```

Comment:
There already exists a class with the name SwedbankPayClient which makes this confusing. Should be renamed to swedbankPayHttpClient.